specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5247 @@
+        "description": {
+          "description": "The service description.",
+          "type": "string"
+        },
+        "kvTags": {
+          "description": "The service tag dictionary. Tags are mutable.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "description": "The service property dictionary. Properties are immutable.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "operationId": {
+          "description": "The ID of the latest asynchronous operation for this service.",
+          "type": "string"
+        },
+        "state": {
+          "description": "The current state of the service.",
+          "enum": [
+            "Transitioning",
+            "Healthy",
+            "Unhealthy",
+            "Failed",
+            "Unschedulable"
+          ],
+          "type": "string",
+          "example": "Healthy",
+          "x-ms-enum": {
+            "name": "WebServiceState",
+            "modelAsString": false
+          }
+        },
+        "createdTime": {
+          "format": "date-time",
+          "description": "The time the service was created.",
+          "type": "string"
+        },
+        "updatedTime": {
+          "format": "date-time",
+          "description": "The time the service was updated.",
+          "type": "string"
+        },
+        "error": {
+          "description": "The error details.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          ]
+        },
+        "computeType": {
+          "description": "The compute environment type for the service.",
+          "enum": [
+            "ACI",
+            "AKS"
+          ],
+          "type": "string",
+          "example": "AKS",
+          "x-ms-enum": {
+            "name": "ComputeEnvironmentType",
+            "modelAsString": true
+          }
+        },
+        "deploymentType": {
+          "description": "The deployment type for the service.",
+          "enum": [
+            "GRPCRealtimeEndpoint",
+            "HttpRealtimeEndpoint",
+            "Batch"
+          ],
+          "type": "string",
+          "example": "HttpRealtimeEndpoint",
+          "x-ms-enum": {
+            "name": "DeploymentType",
+            "modelAsString": false
+          }
+        },
+        "createdBy": {
+          "description": "The individual last responsible for creating or updating the service.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/User"
+            }
+          ]
+        }
+      },
+      "discriminator": "computeType"
+    },
+    "PaginatedServiceList": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning service objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
```

Comment:
please mark properties in here as readonly

Reply:
made changes.